README.md

```diff
@@ -56,7 +56,21 @@ If you using Jest we recommend using [solid-jest](https://github.com/solidjs/sol
 
 See the [docs](https://testing-library.com/docs/preact-testing-library/intro) over at the Testing Library website.
 
-There are two key difference, though:
+There are several key differences, though:
+
+⚠️ The `render` function takes in a function that returns a Solid Component, rather than simply the component itself.
+
+**Preact Testing Library**
```

Comment:
I'd prefer a shorter version using comments instead of headlines:

```js
// With preact-testing-library:
const { container } = render(<Tested component />);

// With solid-testing-library
const { container } = render(() => <TestedComponent />);
```

Reply:
Updated